modules/HomePage/pages/home-page.tsx

```diff
@@ -13,6 +14,17 @@ interface WidgetData {
 }
 
 const HomePage: FC<IPostFeed.IPosts> = ({ data }): ReactElement => {
+  const router = useRouter();
+  useEffect(() => {
+    if (document.cookie.split('=')[0] !== 'user') {
```

Comment:
too naive way. please use cookies APIs

Reply:
You mean a package like cookie-cutter?

Reply:
have you tried to google "cookies API" before asking this question? 😄